--- /Users/Kevin/GitHub/SetupHelper.copy/FileSets/PatchSource/PageSettings.qml.orig	2024-05-15 13:06:53
+++ /Users/Kevin/GitHub/SetupHelper.copy/FileSets/PatchSource/PageSettings.qml	2025-01-24 22:39:59
@@ -192,5 +192,11 @@
 			subpage: Component { PageDebug {} }
 			showAccessLevel: User.AccessService
 		}
+//////// added for PackageManager
+		MbSubMenu
+		{
+			description: qsTr("Package manager")
+			subpage: Component { PageSettingsPackageManager {} }
+		}
 	}
 }
